test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -687,6 +687,42 @@ Describe "TabCompletion" -Tags CI {
             $res.CompletionMatches[1].CompletionText | Should -BeExactly 'dog'
         }
 
+        It "Tab completion for ArgumentCompleter when AST is passed to CompleteInput" {
+            $script = @'
```

Comment:
Is there a reason you need this script string and the script block below instead of calling `ToString` on the Script block?

Reply:
I believe the alternative would be calling `ToString`, and then appending `Test-Completion -TestVal `. If that's preferred I can get that changed.

Reply:
Changed.